new-plans/ir.md

```
@@ -167,37 +157,25 @@ tribute.nat      // 자연수 타입
 
 | Pass 완료 후 | 조건                            |
 | ------------ | ------------------------------- |
-| Resolution   | tribute.var/call 대부분 해소    |
-| Type Check   | tribute.type_var 모두 해소      |
+| Resolution   | 모든 이름 해소됨                |
+| Type Check   | 타입 추론 변수 모두 해소 (front-end에서 처리) |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)